docs/permissions/data-sandboxes-draft.md

```diff
@@ -0,0 +1,269 @@
+
+For an example, see [Data sandboxing: limiting access to columns](https://www.metabase.com/learn/permissions/data-sandboxing-column-permissions).
+
+## Restricting rows in an custom sandbox with user attributes
+
+You can set up an custom sandbox to display different rows to each person depending on their [user attributes](../people-and-groups/managing.md#adding-a-user-attribute). For example, you can display the "Sandboxed Accounts" question with the filter `Plan = "Basic"` for one group, and the filter `Plan = "Premium"` for another group.
+
+1. Make sure you've done all the [prerequisites for custom sandboxes](#prerequisites-for-custom-sandboxes).
+2. Go to the SQL question that will be displayed to the people in the custom sandbox.
+3. Add a [parameterized](../questions/native-editor/sql-parameters.md) `WHERE` clause to your SQL query, such as `{%raw%}[[ WHERE plan = {{ plan_variable }} ]]{%endraw%}`.
+4. Save the SQL question.
+5. Go to **Admin settings** > **Permissions**.
+6. Find the group and table for your custom sandbox.
+7. Open the dropdown under **Data access**.
+8. Click **Edit sandboxed access**.
+9. Scroll down and set **Parameter or variable** to the name of the parameter in your saved SQL question (such as "Plan Variable").
+10. Set the **User attribute** to the **key** of the [user attribute](../people-and-groups/managing.md#adding-a-user-attribute) to be filtered on (such as "User's Plan").
+11. Click **Save**.
+
+### How row restriction works in an custom sandbox
+
+How user attributes, SQL parameters, and custom sandboxes work together to display different rows to different people.
+
+A standard `WHERE` clause filters a table by setting a column to a fixed value:
+```
+WHERE column_name = column_value
+```
+
+In step 2 of the [row restricting configuration](#restricting-rows-in-an-custom-sandbox-with-user-attributes) above, you'll add a SQL variable so that the `WHERE` clause will accept a dynamic value. The [SQL variable type](../questions/native-editor/sql-parameters.md#sql-variable-types) must be text, number, or date.
+```
+WHERE plan = {%raw%}{{ plan_variable }}{%endraw%} 
+```
+
+In steps 9-10 of the [row restricting configuration](#restricting-rows-in-an-custom-sandbox-with-user-attributes) above, you're telling Metabase to map the SQL variable `plan_variable` to a **user attribute key** (such as "User's Plan"):
+```
+WHERE plan = USER_ATTRIBUTE_KEY
+```
+
+Metabase will use the key to look up the **user attribute value** (for example, the "User's Plan" key can map to the values "Basic", "Business", or "Premium"):
+```
+WHERE plan = USER_ATTRIBUTE_VALUE
+```
+
+Metabase replaces the SQL parameter with the specific **user attribute value** (such as "Basic") associated with a sandboxed person. When that person logs into Metabase and uses the sandboxed table, they'll see the query result that uses:
+```
+WHERE plan = "Basic"
+```
+
+## How sandboxing permissions interact with other permissions
```

Comment:
I'm still not sure we need this section. Rather: it's better to just list the scenarios, like you do below. By including this section (its table in particular), we needlessly risk incompletion. What about models? actions? etc.

If we _only_ list the scenarios where things will not respect the sandboxes, we should be good, and it's less info for people to have to read.

Reply:
Yeah, I let this sink in some more. I think I was hanging onto this approach because I spend so much time in troubleshooting-land, where going through each case is very useful! 

Agree that it is unnecessary for the docs :)